cmd/sqlcmd/sqlcmd.go

```diff
@@ -248,10 +247,7 @@ func normalizeFlags(rootCmd *cobra.Command) error {
 			case "default", "readonly":
 				return pflag.NormalizedName(name)
 			default:
-				_, err = normalizeWithError(value, fmt.Errorf("--application-intent must be one of \"default\",\"readonly\" but got \"%s\"", value))
-				if err != nil {
-					fmt.Fprintln(os.Stderr, err)
-				}
+				err = fmt.Errorf("--application-intent must be one of \"default\",\"readonly\" but got \"%s\"", value)
```

Comment:
>"default\",\"readonly\" but got \ [](http://example.com/codeflow?start=59&length=33)

coordinate your merge with Apoorv, as the allowed values need to be constants, not part of the localizable string.

Reply:
We will sync by merging this PR first and then localization PR